at_secondary/at_secondary_server/lib/src/connection/outbound/outbound_message_listener.dart

```diff
@@ -11,22 +11,29 @@ class OutboundMessageListener {
   var logger = AtSignLogger('OutboundMessageListener');
   final _buffer = ByteBuffer(capacity: 10240000);
   late Queue _queue;
+  var _connection;
 
-  OutboundMessageListener(this.outboundClient);
+  OutboundMessageListener(this.outboundClient){
+    this._connection = outboundClient.outboundConnection!;
```

Comment:
Can we refer to connection from outboundClient.outboundConnection and not have a new variable?

Reply:
after tha latest commit, the code now uses outboundClient.outboundConnection instead of a class variable now